src/composables/queries/useTokenListsQuery.ts

```diff
@@ -0,0 +1,35 @@
+import { reactive } from 'vue';
+import { useQuery } from 'vue-query';
+import { UseQueryOptions } from 'react-query/types';
+import QUERY_KEYS from '@/constants/queryKeys';
+import { TokenListMap } from '@/types/TokenList';
+import { tokenListService } from '@/services/token-list/token-list.service';
+
+/**
+ * TYPES
+ */
+type QueryResponse = TokenListMap;
+
+/**
+ * Fetch all token lists, should only happen once.
+ */
+export default function useTokenListsQuery(
+  options: UseQueryOptions<QueryResponse> = {}
+) {
+  const queryKey = reactive(QUERY_KEYS.TokenLists.All);
+
+  const queryFn = async () => {
+    console.log('Fetching tokenLists');
+    return await tokenListService.getAll();
+  };
+
+  const queryOptions = reactive({
+    enabled: true,
+    refetchOnMount: false,
+    refetchOnWindowFocus: false,
+    refetchOnReconnect: false,
```

Comment:
Based on this artice - https://medium.com/in-the-weeds/fetch-a-query-only-once-until-page-refresh-using-react-query-a333d00b86ff

We might also need to set: `staleTime` to something like "24h".

Might also worth exporting this into something like `FETCH_ONCE_OPTIONS`, etc.

Reply:
Done: https://github.com/balancer-labs/frontend-v2/pull/570/commits/8fef34400f5508c8fb79726cc9016e6e10cfa2a4